test/export/test_converter.py

```diff
@@ -16,16 +16,21 @@
 
 
 class TestConverter(TestCase):
-    def _check_equal_ts_ep_converter(self, mod, inp) -> ExportedProgram:
-        ts_model = torch.jit.script(mod)
+    def _check_equal_ts_ep_converter(
+        self, mod, inp, jit_trace=False
+    ) -> ExportedProgram:
+        if not jit_trace:
+            ts_model = torch.jit.script(mod)
```

Comment:
jit.script doesn't run into this issue?
Also I think it might be better if we check both script/trace in this function.

```python
def check(mod, inp):
    script_mod = torch.jit.script(mod)
    convert(script_mod)

    trace_mod = torch.jit.trace(mod)
    convert(trace_mod)
```

Reply:
Right, it seems like when we `jit.script`, it will not result tensor constant but instead constants with primitive types, which are working fine as we have tested for `prim::Constant`. 

There is one test case has issue of `unrecognized type t`. I think that is not relevant, but I will add a `TODO` and we can revisit this in a separate PR.

Reply:
I mean that we should test both in `check`, and then add flags to disable testing for script/trace if they dont work